### ISSUE TITLE
Discover mode

### ISSUE THREAD

**Issue by poproar** [photodude/FOF3-Basic#5]
i noticed that i was unable to discover until i renamed com_fof3basic.xml to fof3basic.xml.

**Comment by photodude** [comment]
That sounds like Joomla may have changed it's behavior (or there is a bug). There should be no issue with the manifest file being named manifest.xml, com_fof3basic.xml, or fof3basic.xml. I tested this about a week ago with no issue with the manifest file being named com_fof3basic.xml.

https://docs.joomla.org/Manifest_files

But on the off chance there is an issue, do you have a preference for manifest naming style? 
`manifest.xml` or `<extension_name>.xml`

**Comment by poproar** [comment]
Not so much a preference. I like the idea of it being manifest.xml if it is only ever read once but I don't remember if that is the case. I think if I learned to use manifest.xml initially I might have felt more confident about what is going on. Also I want to confirm that I just did another test in a new folder
`administrator/components/com_empty`
with a `com_empty.xml` and it did not get discovered but did as soon as I dropped the `com_` prefix.
